src/content/language/patterns.md

```diff
@@ -360,14 +360,14 @@ double calculateArea(Shape shape) => switch (shape) {
 ### Validating incoming JSON
 
 [Map][] and [list][] patterns work well for destructuring key-value pairs in
-JSON data:
+deserialized data, such as data parsed from JSON:
 
 <?code-excerpt "language/lib/patterns/json.dart (json-1)"?>
```

Comment:
Each of these code snippets are extracted from https://github.com/dart-lang/site-www/blob/main/examples/language/lib/patterns/json.dart (as specified by the `<?code-excerpt` instructions).

Can you also update the source in that file to account for these changes?

If you followed the repository [setup instructions](https://github.com/dart-lang/site-www?tab=readme-ov-file#build-the-site), you can then use `dart run dart_site refresh-excerpts` to update these snippets in the Markdown to match the source files. Or you can manually update them to match.